test/hotspot/jtreg/compiler/loopopts/superword/TestReductions.java

```diff
@@ -1597,15 +1609,7 @@ private static long longAddSimple() {
                   IRNode.MUL_REDUCTION_VL, "> 0",
                   IRNode.MUL_VL,           "= 0"}, // Reduction NOT moved out of loop
         applyIfCPUFeatureOr = {"asimd", "true"},
-        applyIf = {"AutoVectorizationOverrideProfitability", "> 0"})
-    // Note: NEON does not support MulVL for auto vectorization. There is
-    //       a scalarized implementation, but that is not profitable for
-    //       auto vectorization in almost all cases, and would not be
-    //       profitable here at any rate.
-    //       Hence, we have to keep the reduction inside the loop, and
-    //       cannot use the MulVL as the vector accumulator.
-    @IR(failOn = IRNode.LOAD_VECTOR_L,
-        applyIf = {"AutoVectorizationOverrideProfitability", "= 0"})
```

Comment:
Same comment about `AutoVectorizationOverrideProfitability=0`: it should not vectorize on any platform.

Also: why did you remove my comment about NEON?